pkg/cmd/server/origin/master.go

```diff
@@ -309,11 +355,17 @@ func (c *MasterConfig) Run(protected []APIInstaller, unprotected []APIInstaller)
 
 	handler = open
 
+	handler = indexAPIPaths(handler)
```

Comment:
Not sure why that's special. We're adding cors headers to all the console paths also. If cors should be special, maybe that filter should move under the API handlers